### ISSUE TITLE
*TEST* Total Cholesterol tests

### ISSUE THREAD

**Issue by richiecroker** [opensafely/codelist-development#74]
https://codelists.opensafely.org/codelist/user/richard-croker/cholesterol-tests/09896c09/
For review/discusson:
Do we want to include _level_ only, and remove _measurement_?

**Comment by brianmackenna** [comment]
Brief discussion between @inglesp @brianmackenna and @richiecroker . We need to further work to understand a) what is the expected deployment of these codes as per NHS data dictionary and other standards. b) what is the actual deployment and usage of these codes

@inglesp is going to produce an analysis of these codes in OpenSAFELY-EMIS to help inform best measure of service restoration. Useful things we thought about including

- Prevalence of each code
- Amount of codes with a value attached e.g. cholesterol 5.5
- Range of values observed
- unit fields?

**Comment by HelenCEBM** [comment]
Just checked this in TPP for 2020 and found:

Code | Name | Total count | Number of records with numeric value > 0 and less than 10
-- | -- | -- | --
1005671000000105 | Serum cholesterol level | 4.52m | 4.47m
1017161000000104 | Plasma total cholesterol level | 31.8k | 31.7k
850981000000101 | Cholesterol level | 11.3k | 3.5k
395153009 | Pre-treatment serum cholesterol level | <100 | 0

**Comment by richiecroker** [comment]
Thanks @HelenCEBM this is very helpful - we should do with the rest of the proposed sentinel tests.  The question for me is whether the 32k plasma cholesterols is all from one lab/area or spread around?  Is there any way we can find out?
Just looking at openpath data (for slightly different timescale), and NDHT did 42k total cholesterol tests in 12 months, so it's not out of the realms of possibility that it's a single lab using odd codes (or actually doing plasma).

**Comment by richiecroker** [comment]
I also wonder if `cholesterol level` tests are from patients self-reported tests.

**Comment by richiecroker** [comment]
(I mean, it's all <1%, but still interesting)

**Comment by HelenCEBM** [comment]
> The question for me is whether the 32k plasma cholesterols is all from one lab/area or spread around? Is there any way we can find out?

This would be easier to do in the next step via study definition

**Comment by richiecroker** [comment]
@brianmackenna  a complementary codelist for numerical values https://codelists.opensafely.org/codelist/user/richard-croker/cholesterol-tests-numerical-value/7dee4fa4/

**Comment by brianmackenna** [comment]
Included codes look good

`850981000000101` : I'm still not sure about excluding this code as yet - Although low in comparitively there are 1000s of blood tests being processed through this way with numerical results. 

At a minimum I think it needs to be explcitly documented in the methodology section that it has been omitted and we can maybe revisit for specific use cases?

**Comment by richiecroker** [comment]
that's true, but only 27% of them (3000) have numerical results, which suggests that it's not used consistently for numerical purposes

**Comment by brianmackenna** [comment]
I think it will depend on the use case. I would favour including and explicitly documenting this but happy to have it left out but we must document explicitly in the methodology so it's obvious an a judgement call can be made for specific use case

(happy with the first list)
